.env.example

```diff
@@ -1,2 +1,3 @@
 ALCHEMY_API_ENDPOINT=
 ETHERSCAN_KEY=
+TENDERLY_KEY=
```

Comment:
are we using this?

Reply:
yes i'm trying to get it to work tbh i've been talking with support for two days now but they're slow in getting to the root of the problem

will move into seperate PR to not mix concerns